testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -55,14 +55,8 @@ do_one_repo () {
   cd "$WORKSPACE/testeng-ci"
   pip install -r requirements/base.txt
 
-  # if present and non-empty, set the PR description from .git/cleanup-python-code-message
-  # this will allow jobs using this script to make custom PR descriptions
-  if [ -s "$repo_dir/.git/cleanup-python-code-message" ]
-  then
-     local message=$(cat "$repo_dir/.git/cleanup-python-code-message")
-  else
-     local message="$(cat <<EOF
-Python code cleanup by the cleanup-python-code Jenkins job.
+  local description="$(cat <<EOF
+feat: Python code cleanup by the cleanup-python-code Jenkins job.
```

Comment:
Some thoughts:

- It would be nice if we could have custom commit messages/PR descriptions that also automatically include the script and package lists.
- This wasn't what I had in mind before, but now that you mention it we probably *do( want to just go ahead and add "commit title" to the job parameters so that people have to actually describe it.
- I think it's rare that we'll want to have different commit and PR titles/bodies.

What if we did this?

- Add commit/PR title as a single-line field, with a link to the conventional-commits doc. (And make it required, if that's something we can enforce in the UI.)
- Add commit/PR body as an optional multi-line field
- Body is constructed of: Commit body (if provided), contents of `.git/cleanup-python-code-commit-message` (if present), and finally the autogenerated information here about what job created the PR and what scripts and packages were involved
- Commit is just commit title + that body (params already distinguish title vs. body of PR)

...and then I think we end up in a better situation for people who are looking at these PRs and commits out of context.

Reply:
Do we really want the commit to include the full body? That will be a lot of information

Reply:
I don't think it would be *too* too big. I'll often write large commit messages, but I just make sure to do it "pyramid style", with the most concise and important parts up at the top and the most incidental and expanded parts at the bottom -- people don't have to read the whole thing, but it's there if they need it. In our case, I don't think it would run to more than 10-20 lines, max (unless there's a huge script, I guess!)

Reply:
The script we are running is already well over 10-20 lines.